### ISSUE TITLE
When installing dev tools: The requirement ('Unidecode>=0.04.16') is not allowed by your [versions] constraint (0.04.1)

### ISSUE THREAD

**Issue by jimfulton** [plone/Installers-UnifiedInstaller#78]
When following instructions here: 
https://docs.plone.org/manage/installing/installation.html#installing-plone-using-the-unified-unix-installer

I tried installing dev tools:

```
cd ~/Plone/zinstance
bin/buildout -c develop.cfg
```

And got:

```
While:
  Installing mrbob.
Error: The requirement ('Unidecode>=0.04.16') is not allowed by your [versions] constraint (0.04.1)
```

I'm running Ubunto 18.04.

I don't need a solution myself, just reporting...

**Comment by adaugherity** [comment]
I encountered this myself (plone/bobtemplates.plone#337) and resolved it by pinning `bobtemplates.plone = 3.4.1` in the [versions] section of buildout.cfg.  Version 3.4.2 added a dependency on python-slugify, which wants the newer unidecode.

The 5.0.x unified installer pins bobtemplates.plone to 1.0.5; 5.1.0 pinned it to 2.0 but that pin was removed with 5.1.1.

A quick solution would be to restore this bobtemplate.plone version pin to the 5.1.x branch and add one to the 4.3.x branch, but the better long-term solution (at least for 5.1.x) would be to fix the Plone unicode infrastructure to work with newer versions (i.e. plone/Products.CMFPlone#2599).
